src/views/Home/Home.jsx

```diff
@@ -13,7 +13,46 @@ function Home() {
 	*/
 	return (
 		<div className={styles.wrapper}>
-			<p>Home</p>
+			<div className={styles.contentBoxOne}>
+				<h1> Welcome to the brains</h1>
+				<h3>
+					Welcome to ask the brains, where we answer your questions to
+					sell you the perfect virtual pet
+				</h3>
+			</div>
+			<div className={styles.primaryContentContainer}>
+				<div className={styles.contentBoxTwo}>
+					<h2>We do AI</h2>
+					<h2 className={styles.purpleH2}>(via ChatGPT)</h2>
+					<p>
+						Ask us a question, we'll secretly pass it on to ChatGPT
+						and will return the result pretending that we generated
+						it ourselves
+					</p>
+				</div>
+				<div className={styles.contentBoxThree}>
+					<img
+						className={styles.homeIll}
+						src="src/views/Home/img/home_ill.png"
```

Comment:
Looks good. Only thing is the image needs a a tweak to get it to load in the build.

```jsx
import homeImgSrc from "src/views/Home/img/home_ill.png";

function Home() {
  return <...>
    <img src={homeImgSrc} ... />
  </...>;
}
```

Alternatively the image can be moved to a new directory in the root of the project called "public", in which case its URL path is always the same as the file path relative to the folder. For example if it's in public/foo/bar and it's called foobar.png, the src should be "/public/foo/bar/foobar.png".

Reply:
Yeah, I just put the image  in the existing component file for now, because I didn't want to interact heavily with other files and cause merge conflicts

Reply:
I have created /public/ and moved the file @hedgehog125

Reply:
That makes sense. Are you ok with implementing the first solution then or should I try and explain it on a call?